Settings.py

```diff
@@ -112,9 +112,11 @@ def get_settings_string(self):
                     i_bits.extend(item_bits)
                 i_bits.extend(terminal)
 
-            # pad it
-            i_bits += [0] * ( setting.bitwidth - len(i_bits) )
+            else:
+                # pad it
+                i_bits += [0] * ( setting.bitwidth - len(i_bits) )
```

Comment:
Just curious why this was changed to only occur when the setting is not a list. We have a few other lists in settings that this would also affect.

Reply:
I think that change was from the previous GUI. It's no longer necessary and has been reverted. Thanks for the catch.